bin/bash-lint

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+base_dir="$(dirname "$0")"/..
+
+for f in "$base_dir"/bin/*; do shellcheck "$base_dir/bin/$f"; done
```

Comment:
Kinda sucks to have to copy this all over the place. Can shellcheck either automatically check an entire directory or could we maybe just encode this directly in the pom? (Or is there a variant that can be used that does support a more project-based approach that handles entire directories?)